druid/examples/focus.rs

```diff
@@ -0,0 +1,135 @@
+// Copyright 2020 The Druid Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Demonstrates focus and focus scope widgets
+
+pub use druid::theme;
+use druid::widget::prelude::*;
+use druid::widget::{Button, Container, Flex, Focus, FocusScope, Label, WidgetExt};
+
+use druid::{AppLauncher, Color, Data, LocalizedString, Point, Rect, WidgetPod, WindowDesc};
+
+pub struct FocusDecorator<T> {
```

Comment:
In examples I would be generous with documentation; for instance here I would say something like, "A container that draws a border around its child when focused.".

Reply:
ok